pkg/cmd/tknpac/bootstrap/bootstrap.go

```diff
@@ -261,17 +264,37 @@ func DetectPacInstallation(ctx context.Context, wantedNS string, run *params.Run
 		return installed, "", fmt.Errorf("could not detect Pipelines as Code configmap in %s namespace : %w, please reinstall", wantedNS, err)
 	}
 
-	cms, err := run.Clients.Kube.CoreV1().ConfigMaps("").List(ctx, metav1.ListOptions{
-		LabelSelector: configMapPacLabel,
-	})
+	cm, err := getConfigMap(ctx, run)
 	if err == nil {
-		for _, cm := range cms.Items {
-			if cm.Name == infoConfigMap {
-				return installed, cm.Namespace, nil
+		return installed, cm.Namespace, nil
+	}
+	return installed, "", fmt.Errorf("could not detect Pipelines as Code configmap on the cluster, please specify the namespace in which pac is installed")
+}
+
+func getConfigMap(ctx context.Context, run *params.Run) (*corev1.ConfigMap, error) {
+	var (
+		err       error
+		configMap *corev1.ConfigMap
+	)
+	for _, n := range defaultNamespaces {
+		configMap, err = run.Clients.Kube.CoreV1().ConfigMaps(n).Get(ctx, infoConfigMap, metav1.GetOptions{})
+		if err != nil {
+			if kapierror.IsNotFound(err) {
+				continue
 			}
+			if strings.Contains(err.Error(), fmt.Sprintf(`cannot get resource "configmaps" in API group "" in the namespace "%s"`, n)) {
+				continue
+			}
+			return nil, err
+		}
+		if configMap != nil {
+			break
 		}
 	}
-	return installed, "", fmt.Errorf("could not detect Pipelines as Code configmap on the cluster, please reinstall")
+	if configMap == nil {
+		return nil, fmt.Errorf("ConfigMap not found in default namespaces")
```

Comment:
```suggestion
		return nil, fmt.Errorf("ConfigMap not found in default namespaces("openshift-pipelines", "pipelines-as-code")
```
or something

because the word `default namespaces` conflict with actual k8s/openshift default ns